crawler/crawler.conf

```diff
@@ -2,21 +2,3 @@
     #enabled_plugins = os_container, cpu_container
 
 [ crawlers ]
```

Comment:
Can you please bring this list back here, it doesn't affect which plugins get loaded, that is controlled by enabled_plugins in the above section